test/Address.test.js

```diff
@@ -0,0 +1,22 @@
+const AddressImpl = artifacts.require('AddressImpl');
+const SimpleToken = artifacts.require('SimpleToken');
+
+require('chai')
+  .should();
+
+contract('Address', function ([_, creator]) {
+  beforeEach(async function () {
+    this.mock = await AddressImpl.new();
+  });
+
+  it('should return false for account address', async function () {
+    const isContract = await this.mock.isContract(creator);
+    isContract.should.equal(false);
+  });
+
+  it('should return true for contract address', async function () {
+    const contract = await SimpleToken.new({ from: creator });
```

Comment:
This `from: creator` isn't really needed, why is why I suggested the rename, and removing this bit (to avoid unnecessary confusion about a creator address) :)